force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +113,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String evenOrOddNumber = null;
+        if(a != null) {
+        if (Math.mod(a, 2) == 0){
```

Comment:
What happens if we directly return the result of `Math.mod(a, 2) == 0`

Is there an easier to read way to check if a is null before we do our calculations?